examples/04_scalability.py

```diff
@@ -44,7 +47,7 @@ def wrapped_func(*args, **kwargs):
 # --------------------------------
 #
 # We first download the dataset:
-from dirty_cat.datasets import fetch_traffic_violations
+from dirty_cat.datasets.fetching import fetch_traffic_violations
```

Comment:
We prefer to avoid importing from subpackages; here I think that it's better to directly import from datasets.